kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-forms/kie-wb-common-stunner-forms-backend/src/test/java/org/kie/workbench/common/stunner/forms/backend/service/impl/FormDefinitionGeneratorImplTest.java

```
@@ -205,6 +208,22 @@ public void init() throws Exception {
         when(diagramPath.toURI()).thenReturn("default:///src/main/resources" + PROCESS_PATH);
     }
 
+    private void loadClass() {
+        try {
+            when(moduleClassLoader.loadClass(anyString())).thenAnswer(invocation -> Object.class);
+        } catch (ClassNotFoundException e) {
+            // Do nothing.
```

It is probably not the best idea letting empty catch blocks, or is it?

---

I can't imagine that Object class won't be found. But if won't we will get failing test, runtime not affected. I think it is pretty save to have it like this. Do you have better ideas?

---

Hi @LuboTerifaj, is it OK to you?

---

I see. I would let it be as it is now. Thanks!